localisation/00_wc_random_artifact_names.csv

```diff
@@ -21,60 +37,60 @@ artifact_rare_origin;Fables tell that [GetArtifactRareOrigin].;Des fables disent
+;;;;;;;;;;;;;;x
+artifact_epic_origin_1;[GetGrammaticalNumberThisThese] [This.GetName] fell from the sky;ce [This.GetName] est tomb� du ciel;;;;;;;;;;;;x
+artifact_epic_origin_2;the origins of [GetGrammaticalNumberThisThese] [This.GetName] are unknown;les origines de ce [This.GetName] sont inconnues;;;;;;;;;;;;x
+artifact_epic_origin_3;[GetGrammaticalNumberThisThese] [This.GetName] [GetGrammaticalNumberWasWere] the spark that ignited a full-scale war between neighbouring realms, as they vied to acquire [GetGrammaticalNumberItThem];ce [This.GetName] a �t� l'�tincelle qui a d�clench� une guerre � grande �chelle entre des royaumes voisins, alors qu'ils rivalisaient pour l'acqu�rir;;;;;;;;;;;;x
+artifact_epic_origin_4;[GetGrammaticalNumberThisThese] [This.GetName] [GetGrammaticalNumberIsAre] the only remaining proof of a mystic creature's existence, which died a long time ago;ce [This.GetName] est la seule preuve restante de l'existence d'une cr�ature mystique morte il y a longtemps;;;;;;;;;;;;x
+artifact_epic_origin_5;[GetGrammaticalNumberThisThese] [This.GetName] [GetGrammaticalNumberWasWere] sold once for 40000;ce [This.GetName] a �t� vendu une fois pour 40 000 pi�ces d'or;;;;;;;;;;;;x
+artifact_epic_origin_6;the original owner of [GetGrammaticalNumberThisThese] [This.GetName] spent the last years of his life traveling the world in search of esoteric secrets;le premier propri�taire de ce [This.GetName] a pass� les derni�res ann�es de sa vie � parcourir le monde � la recherche de secrets �sot�riques;;;;;;;;;;;;x
+artifact_epic_origin_7;sometimes [GetGrammaticalNumberThisThese] [This.GetName] glow[GetGrammaticalNumberEndingSingularS] with the essence of several lives that ended too soon;parfois ce [This.GetName] brille avec l'essence de plusieurs vies qui se sont termin�es trop t�t;;;;;;;;;;;;x
+artifact_epic_origin_8;when you gaze too long at [GetGrammaticalNumberThisThese] [This.GetName], [GetGrammaticalNumberItThey] also gaze[GetGrammaticalNumberEndingSingularS] back at you;quand vous regardez trop longtemps ce [This.GetName], il vous regarde aussi en retour;;;;;;;;;;;;x
+artifact_epic_origin_9;[GetGrammaticalNumberThisThese] [This.GetName] whisper[GetGrammaticalNumberEndingSingularS] [GetGrammaticalNumberItsTheir] original owner's name from realms beyond;ce [This.GetName] chuchote le nom de son propri�taire d'origine depuis des royaumes lointains;;;;;;;;;;;;x
+artifact_epic_origin_10;once upon a time a wise hermit has tried to understand the power of [GetGrammaticalNumberThisThese] [This.GetName], but has been driven mad by it;un sage ermite a essay� de comprendre le pouvoir de ce [This.GetName], mais en a �t� rendu fou;;;;;;;;;;;;x
```

Comment:
Took out has, changed the second has to was.
```suggestion
artifact_epic_origin_10;once upon a time a wise hermit tried to understand the power of [GetGrammaticalNumberThisThese] [This.GetName], but was been driven mad by it;un sage ermite a essay� de comprendre le pouvoir de ce [This.GetName], mais en a �t� rendu fou;;;;;;;;;;;;x
```

Reply:
Done